docs/ai-coder/ai-bridge/index.md

```diff
@@ -2,7 +2,7 @@
 
 ![AI bridge diagram](../../images/aibridge/aibridge_diagram.png)
 
-AI Bridge is a smart proxy for AI. It acts as a man-in-the-middle between your users' coding agents / IDEs
+AI Bridge is a smart gateway for AI. It acts as an intermediary between your users' coding agents / IDEs
```

Comment:
@matifali added some drive-by improvements to the wording
It's not in scope of the other changes, but I didn't want to open another PR and trouble you guys for another review.